ospf6: add the ability of specifying router-id/area-id in no debug ospf6

caracall opened the pull request:
Add the ability to specify the router-id/area-id when deleting the debug
ospf6 configuration.

The new commands are as follow:

no debug ospf6 border-routers router-id [A.B.C.D]
no debug ospf6 border-routers area-id [A.B.C.D]

Signed-off-by: Ahmad Caracalli <ahmad.caracalli@6wind.com>

idryzhov reviewed (approved):
I suppose https://github.com/FRRouting/frr/pull/9739 can be closed now?

pguibert6WIND commented:
> I suppose #9739 can be closed now?

yes

idryzhov commented:
@caracall thanks for the update, but we don't document "no" forms of the commands. Please, remove "no" and update the descriptions accordingly.

caracall commented:
Thanks @idryzhov for the follow up, I've updated the main command in the documentation with the options that we've added to the code. Hope it fits this time.
